lib/reset.js

```diff
@@ -11,7 +11,7 @@ function reset (state) {
   })
 
   .then(function () {
-    return state.db.destroy
+    return state.db.destroy()
```

Comment:
You're most welcome, Gregor.  If I had a dollar every time I did that, I'd be retired already! :)